frontend/src/main/pages/HomePage.js

```diff
@@ -16,25 +16,44 @@ export default function HomePage() {
 
     const greeting = getPartOfDayGreeting();
 
-    if (!currentUser || !currentUser.loggedIn) {
-        return (
-            <div data-testid={"HomePage-main-div"}>
-                <BasicLayout>
-                    <h1 data-testid="homePage-title" style={{ fontSize: "75px", borderRadius: "7px", backgroundColor: "white", opacity: ".9" }} className="text-center border-0 my-3">
-                        {greeting}, cgaucho
-                    </h1>
-                </BasicLayout>
-            </div>
-        );
-    }
-
-    // Stryker disable all : TODO: restructure this code to avoid the need for this disable
+    const renderGreeting = (username) => (
+        <h1 
+            data-testid="homePage-title" 
+            style={{ 
+                fontSize: "75px", 
+                borderRadius: "7px", 
+                backgroundColor: "white", 
+                opacity: ".9" 
+            }} 
+            className="text-center border-0 my-3"
+        >
+            {greeting}, {username}
+        </h1>
+    );
+
+    const renderInfoSection = () => (
+        <div data-testid="info-section" style={{ padding: "20px", borderRadius: "7px", marginTop: "20px" }}>
+            <h2>About Organic</h2>
+            <p>
+            Organic is a platform designed to equip students and instructors with a comprehensive suite of tools for efficiently managing GitHub organizations associated with computer science courses.
+            </p>
+            <p>
+            Users can input detailed course information, including the course name, term, start/end dates, and GitHub organization. Additionally, users can specify their institution's name and abbreviation, as well as the academic quarter or semester they are in. 
+            </p>
+            <img 
+                src="https://www.startyourlab.com/img/setup/github-organization.svg" 
+                alt="CS156 Project" 
+                style={{ width: "100%", borderRadius: "7px", marginTop: "20px" }}
+            />
+        </div>
+    );
+   
+
     return (
         <div data-testid={"HomePage-main-div"}>
             <BasicLayout>
-                <h1 data-testid="homePage-title" style={{ fontSize: "75px", borderRadius: "7px", backgroundColor: "white", opacity: ".9" }} className="text-center border-0 my-3">
-                    {greeting}, {currentUser.root.user.githubLogin}
-                </h1>
+                {renderGreeting(currentUser.loggedIn ? currentUser.root.user.githubLogin : "cgaucho")}
```

Comment:
This is much better!  But, I'd suggest that we drop the "cgaucho" and just go with nothing.  I'm not sure who thought cgaucho was a good choice here; that's literally a "generic placeholder for a github id".  Greeting someone with that that hasn't logged in yet is just ... weird.
```suggestion
                {renderGreeting(currentUser.loggedIn ? currentUser.root.user.githubLogin : "")}
```